Tests/ApolloCodegenTests/ApolloSchemaInternalTests.swift

```diff
@@ -27,13 +27,5 @@ class ApolloSchemaInternalTests: XCTestCase {
     let postType = try schema.getType(named: "Post")
     XCTAssertEqual(postType?.name, "Post")
   }
-
-  func testConfiguration_usingOnlyOutputFolders_shouldGenerateCompatibleFilenames() {
```

Comment:
Moved into the public tests file since `ApolloCodegenConfiguration` now publicly exposes more of its properties.